packages/Molecules/Form/fields/fieldRplmarkup.vue

```diff
@@ -1,12 +1,18 @@
 <template>
-  <div class="rpl-markup" v-html="schema.markup"></div>
+  <rpl-wysiwyg class="rpl-markup" :plugins="this.schema.plugins" :html="markup" />
```

Comment:
plugins can be passed through RplMarkup using schema.plugins. This way you can choose exactly which markup to support